contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -82,6 +82,9 @@ abstract contract InitializableAbstractStrategy is Initializable, Governable {
      * @param _config The platform and OToken vault addresses
      */
     constructor(BaseStrategyConfig memory _config) {
```

Comment:
Not sure why we don't call the `Governable()` constructor here?

Reply:
though all that does is set the governor to msg.sender. I guess it is ok like it is

Reply:
The `Governable` constructor is still executed even if the contract contracts inheriting `Governable` don't explicitly call `Governable()`.

As an example, contracts `B`, `C` and `D` in the below all have `setting` equal to 1.

```Solidity
contract A {
    uint256 public setting;

    constructor() {
        setting = 1;
    }
}

contract B is A {
    constructor() {
    }
}

contract C is A {
    constructor() A() {
    }
}

contract D is A {
}
```